modules/imgproc/src/contours_new.cpp

```diff
@@ -627,37 +619,47 @@ bool ContourScanner_::findNext()
 
 //==============================================================================
 
-void cv::findContours(InputArray _image, OutputArrayOfArrays _contours, OutputArray _hierarchy, int mode, int method, Point offset)
+void cv::findContours(InputArray _image,
+                      OutputArrayOfArrays _contours,
+                      OutputArray _hierarchy,
+                      int mode,
+                      int method,
+                      Point offset)
 {
     CV_INSTRUMENT_REGION();
 
     // TODO: remove this block in future
     if (method == 5 /*CV_LINK_RUNS*/)
     {
-        CV_LOG_ONCE_WARNING(NULL, "LINK_RUNS mode has been extracted to separate function: cv::findContoursLinkRuns. "
+        CV_LOG_ONCE_WARNING(
+            NULL,
+            "LINK_RUNS mode has been extracted to separate function: cv::findContoursLinkRuns. "
             "Calling through cv::findContours will be removed in future.");
         CV_CheckTrue(!_hierarchy.needed() || mode == RETR_CCOMP,
-            "LINK_RUNS mode supports only simplified hierarchy output (mode=RETR_CCOMP)");
+                     "LINK_RUNS mode supports only simplified hierarchy output (mode=RETR_CCOMP)");
         findContoursLinkRuns(_image, _contours, _hierarchy);
         return;
     }
 
     // TODO: need enum value, need way to return contour starting points with chain codes
     if (method == 0 /*CV_CHAIN_CODE*/)
     {
-        CV_LOG_ONCE_WARNING(NULL, "Chain code output is an experimental feature and might change in future!");
+        CV_LOG_ONCE_WARNING(
+            NULL,
+            "Chain code output is an experimental feature and might change in future!");
     }
 
     // Sanity check: output must be of type vector<vector<Point>>
     CV_Assert((_contours.kind() == _InputArray::STD_VECTOR_VECTOR)
-        || (_contours.kind() == _InputArray::STD_VECTOR_MAT)
-        || (_contours.kind() == _InputArray::STD_VECTOR_UMAT));
+              || (_contours.kind() == _InputArray::STD_VECTOR_MAT)
+              || (_contours.kind() == _InputArray::STD_VECTOR_UMAT));
```

Comment:
`_contours.kind()` should be aligned to have readable code.

Reply:
Here we are seeing same situation as in previous comments. We can move this expression outside, but macro would loose its meaning. Or we can add this macro to whitespace-preserve list. I'd prefer to create new check macro to cover these cases.